src/extension/devtools/panel.html

```diff
@@ -16,16 +16,14 @@
         height: 100vh;
       }
     </style>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
```

Comment:
This might not be a good idea due to GDPR (although this is not a website).
Can we include the fonts?

Reply:
We used Google fonts before: https://github.com/apollographql/apollo-client-devtools/blob/0e90a112a9ed24e0368536b9c4c275be3358eec7/src/extension/devtools/panel.html#L19-L21

This doesn't connect to `fonts.gstatic.com` though, so I'll remove for now to be safe. I assume because we were already using Google fonts that we'd be ok to use it for the new fonts?

Reply:
[515c692](https://github.com/apollographql/apollo-client-devtools/pull/1180/commits/515c6925aa73a8f2b655157c26ef7470449d1ea7)

Reply:
Yeah, probably a general follow-up question: do we want to use google fonts or just ship one? :)

Reply:
I asked the design team about using google fonts and it seemed to be ok to use. We can always change later if need be 🙂